File: u3u2/Assets/Scripts/app/net/s2c/GCEqpGemSet.cs

using System;
namespace app.net
{
/**
 * 镶嵌宝石结果
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCEqpGemSet :BaseMessage
{
	/** 装备UUID */
	private string equipUuid;
	/** 要镶嵌的宝石模板Id */
	private int gemItemId;
	/** 镶嵌上的宝石模板Id，为0表示失败扣除 */
	private int finalGemItemId;

	public GCEqpGemSet ()
	{
	}

	protected override void ReadImpl()
	{
	// 装备UUID
	string _equipUuid = ReadString();
	// 要镶嵌的宝石模板Id
	int _gemItemId = ReadInt();
	// 镶嵌上的宝石模板Id，为0表示失败扣除
	int _finalGemItemId = ReadInt();


		this.equipUuid = _equipUuid;
		this.gemItemId = _gemItemId;
		this.finalGemItemId = _finalGemItemId;
	}
	
	protected override void WriteImpl()
    {
        return;
    }
	
	public override short GetMessageType() 
	{
		return (short)MessageType.GC_EQP_GEM_SET;
	}
	
	public override string getEventType()
	{
		return EquipGCHandler.GCEqpGemSetEvent;
	}
	

	public string getEquipUuid(){
		return equipUuid;
	}
		

	public int getGemItemId(){
		return gemItemId;
	}
		

	public int getFinalGemItemId(){
		return finalGemItemId;
	}
		

}
}

File: server/当前版本/server/zlj1/game_server/src/com/imop/lj/gameserver/equip/msg/GCEqpGemSet.java
package com.imop.lj.gameserver.equip.msg;

import com.imop.lj.core.msg.MessageType;
import com.imop.lj.gameserver.common.msg.GCMessage;
/**
 * 镶嵌宝石结果
 *
 * @author CodeGenerator, don't modify this file please.
 */
public class GCEqpGemSet extends GCMessage{
	
	/** 镶嵌宝石结果 1成功 2失败 */
	private int result;

	public GCEqpGemSet (){
	}
	
	public GCEqpGemSet (
			int result ){
			this.result = result;
	}

	@Override
	protected boolean readImpl() {

	// 镶嵌宝石结果 1成功 2失败
	int _result = readInteger();
	//end



		this.result = _result;
		return true;
	}
	
	@Override
	protected boolean writeImpl() {

	// 镶嵌宝石结果 1成功 2失败
	writeInteger(result);


		return true;
	}
	
	@Override
	public short getType() {
		return MessageType.GC_EQP_GEM_SET;
	}
	
	@Override
	public String getTypeName() {
		return "GC_EQP_GEM_SET";
	}

	public int getResult(){
		return result;
	}
		
	public void setResult(int result){
		this.result = result;
	}
}